### DISCUSSION TITLE
Welcome to Cantus Ultimus Discussions!

### DISCUSSION THREAD

**Discussion by fujinaga** [DDMAL/cantus#383] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fujinaga** [comment]
This is my first comment on any Github Discussions ever!
To commemorate this occasion, I would like to share a great document from Alessandra Ignesti for locating IIIF manifest on websites hosting IIIF images:
[IIIF manifest_cheat sheet.docx.pdf](https://github.com/DDMAL/cantus/files/6816907/IIIF.manifest_cheat.sheet.docx.pdf)

**Reply by napulen** [reply]
This is great! Thank you, @fujinaga

This will be very helpful for keeping the potential manuscripts of Cantus Ultimus updated.

Just for completeness, the partial spreadsheet that Alessandra compiled a few weeks ago is currently available here: https://dev-cantus.simssa.ca/manifests/

Maybe we should change the layout/formatting of this table, or how it is accessed from the website. For now, it's there because it's very useful when playing around with the website.

**Reply by napulen** [reply]
These are also my first comments ever on Github Discussions, by the way :fire:
